src/Exception/AnnotationNotDefinedException.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Opstalent\CrudBundle\Exception;
+
+use LogicException;
+use Throwable;
+
+/**
+ * Class AnnotationNotDefinedException
+ * @author Szymon Kunowski <szymon.kunowski@gmail.com>
+ * @package Opstalent\CrudBundle\Exception
+ */
+class AnnotationNotDefinedException extends LogicException implements Exception
+{
+    const DEFAULT_MESSAGE = "Annotation not defined properly. 
+        Please check documentation on https://github.com/opstalent/crud/blob/master/README.md";
```

Comment:
is this line needed?

Reply:
no but constructor looks better that way

Reply:
I mean this tet about docs. IMO shoud be `const DEFAULT_MESSAGE = "Annotation not defined properly.";`

Reply:
ok i was thinking it would be nice if i forgot to add annotation error tell me where to find doc